command/core.py

```diff
@@ -1,11 +1,12 @@
 """Calls the appropriate handler depending on the event data."""
-from command.commands.user import UserCommand
-from command.commands.token import TokenCommand
 import command.util as util
-from model.user import User
-from interface.slack import SlackAPIError
 import logging
+
+from command.commands.user import UserCommand
+from command.commands.token import TokenCommand
```

Comment:
Might wanna change this to `from command.commands import UserCommand, TokenComand`, or something even simpler.

Reply:
I considered that, but the `TokenCommand` also exports the command configuration class, and I imagine other commands might do that in the future as well, so `__init__` for `command.commands` could become quite cluttered. But if we're okay with that, I can add that.